guides/release/components/built-in-components.md

```diff
@@ -21,9 +21,27 @@ Will become:
 <label for="site">Ember Question</label>
 <input id="site" type="text" value="How do text fields work?"/>
 ```
+### Naming with aria-label
 
-You can pass the following standard `<input>` attributes within the input
-helper:
+The `aria-label` property enables developers to name an element with a string that is not visually rendered.
```

Comment:
@rwjblue has some feedback on when to use property vs attribute in the prose. I mix them up all the time and I'm not sure which is correct in these examples.